packages/teams-js/src/public/media.ts

```diff
@@ -412,30 +410,38 @@
      * @param mediaEvent indicates what the event that needs to be signaled to the host client
```

Comment:
>isSupported [](http://example.com/codeflow?start=10&length=11)

this should be !isSupported

Reply:
(and lines 415 and 416 could be combined into:

if (!isMobileApiSupported(nonFullScreenVideoModeAPISupportVersion))